app/src/main/java/com/codesoom/assignment/application/UserCommandService.java

```diff
@@ -0,0 +1,43 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserRequest;
+import com.github.dozermapper.core.DozerBeanMapperBuilder;
+import com.github.dozermapper.core.Mapper;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserCommandService {
+    private final UserRepository userRepository;
+    private final Mapper mapper = DozerBeanMapperBuilder.buildDefault();
+
+    public UserCommandService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public User createUser(UserRequest userRequest) {
+        User user = mapper.map(userRequest, User.class);
+        return userRepository.save(user);
+    }
+
+    public User updateUser(Long id, UserRequest userRequest) {
+        User findUser = userRepository.findById(id).orElseThrow(UserNotFoundException::new);
```

Comment:
한 줄에 `.`이 하나씩만 오도록 적절히 개행해 보세요.
```suggestion
        User findUser = userRepository.findById(id)
              .orElseThrow(UserNotFoundException::new);
```

Reply:
넵 !